plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -61,10 +61,10 @@ func calculateResourceOccupancy(pod *api.Pod, node api.Node, nodeInfo *scheduler
 	cpuScore := calculateScore(totalMilliCPU, capacityMilliCPU, node.Name)
 	memoryScore := calculateScore(totalMemory, capacityMemory, node.Name)
 	glog.V(10).Infof(
-		"%v -> %v: Least Requested Priority, Absolute/Requested: (%d, %d) / (%d, %d) Score: (%d, %d)",
+		"%v -> %v: Least Requested Priority, Absolute/Requested(CPU:millicores, memory:bytes): (%d, %d) / (%d, %d) Score: (%d, %d)",
```

Comment:
This would be even clearer as

`"%v -> %v: Least Requested Priority, capacity %d millicores %d memory bytes, total request %d millicores %d memory bytes, score %d CPU %d memory", pod.Name, node.Name, capacityMilliCPU, capacityMemory, totalMilliCPU, totalMemory, cpuScore, memoryScore`